battle.go

```diff
@@ -177,18 +177,24 @@ func (b *Battle) SimulateRound() []Transaction {
 					}
 				}
 				if !anyAlive {
```

Comment:
`anyAlive` is false when all of a *party*'s Pokemon have fainted right? shouldnt we instead check to ensure that all of one team's Pokemon have fainted? (ex. multi battle where Trainer 1 and Trainer 2 form one team, all of Trainer 1's pokemon are knocked out but Trainer 2 still has pokemon)

Reply:
We can do that when we actually do double battles.